drupal-timeline.json

```diff
@@ -80,6 +80,16 @@
     "description": "A major language renewal comes with the new Zend Engine 3 ([PHPNG](https://wiki.php.net/phpng)), It’s also twice as fast as the previous PHP 5.6 release.",
     "comment": ""
   },
+  {
+    "priority": 3,
+    "date": "2016-05-14T00:23:00+00:00",
+    "title": "Drupal Diversity & Inclusion Founded",
+    "username": "sparklingrobots",
+    "avatar": "https://lh3.googleusercontent.com/5yLl0Kt_eNk1widEvAN_WlR-ZsLXOh4e3blA7NDJNzYVFk8CcYXUAvlwgJGA4uJ8O49bRHkayARWDxnX_0vYtezqmsRGdDBYXB-j8B_O7et5Jsc1Dlaq3Owkvm9zczh_Q5i28CDmJ189diPj0ahj5dP3Wbe3dLUwhYHneHfkUo6uNm6ov8Vzu8rsdPJTl7A7ddCM61KaxDQq361oh80mM1BiHTF6NTnC5-IAWo-BKHligpt_t_OAyTao99_ddkoIYNJPgk4RWj6xrDD2DXutTQjJik7fpaVxGSimPGbkiOVGghgJHNGFUc6eC52rYPicAmdZJM23GBjg2ptufVusE1LxYkUEbVsApKASIHBXUepmqKg8pHx2HvntOeRMqHzvyCWjWWWn-jWf1l-PuGyM0n3DdFbwRTesfgB9AL0jsqTuT9YKofPEBQjN-z5QcK6fdtV15vxCIGBt2Fvw8jmUvt1TTNnZspg_D22MsLo-U6DqoytlnK__oi_pQLmUTiQp0ol82mMhWz1wh_hcNbPYgjjnYNRrwhtH7vEr9G9LABeR7QdWKzZNQfQ3k3SUzxtf9fm5wlbzFDdX0re4lB6hNA5ev4oENZVrko4x1LT051yYuvqu3pC2hSzq6aLu7FXQIUI834a7tkNOFldgSK4AO0LndJKM8BfohJA_J8I1pxkMW7D_Un_6b0F5flOfRbg=w1383-h782-no?authuser=0",
+    "href": "https://www.drupaldiversity.com",
+    "description": "After a session on diversity held by Nikki Stevens (https://www.drupal.org/u/drnikki) and Karyn Cassio (https://www.drupal.org/u/techgirlgeek), Drupal Diversity & Inclusion was formed to lead diversity & inclusion efforts in the Drupal Community.",
```

Comment:
Suggested edited description:

After a DrupalCon NA session on diversity led by Nikki Stevens (https://www.drupal.org/u/drnikki) and Karyn Cassio (https://www.drupal.org/u/techgirlgeek), Drupal Diversity & Inclusion (https://drupaldiversity.com) was formed by @drnikki and members of the community to advocate for diversity, equity, and inclusion.

Reply:
These seem like solid improvements to me. Looks like the @ notation for usernames is not used elsewhere in the file, maybe it should just be `...formed by drnikki...` for consistency ?

DDI has been such an important addition to the community. Thanks to you both for working on adding it to this timeline.

Reply:
That makes sense. Thanks, @greggles!

Reply:
Hi, y'all--I modified the text & updated the image link. Hoping this one works better.

Reply:
@rubyji & @greggles if this new version looks good to you, I'll resolve this review. Thanks!

Reply:
Yep, looks good to me.

Reply:
Looks great. Thank you!